mmv1/products/redis/api.yaml

```diff
@@ -394,3 +394,57 @@ objects:
           - :READ_REPLICAS_DISABLED
           - :READ_REPLICAS_ENABLED
         input: true
+      - !ruby/object:Api::Type::NestedObject
+        name: persistenceConfig
+        description: Configuration of the persistence functionality.
+        min_version: beta
+        allow_empty_object: true
+        send_empty_value: true
+        properties:
+        - !ruby/object:Api::Type::Enum
+          name: persistenceMode
+          min_version: beta
+          description: |
+            Optional. 
+            Controls whether Persistence features are enabled. 
+            If not provided, the existing value will be used.
+            - PERSISTENCE_MODE_UNSPECIFIED: Not set.
+            - DISABLED: Persistence is disabled for the instance, and any existing snapshots are deleted.
+            - RDB: RDB based Persistence is enabled.
+          values:
+            - :PERSISTENCE_MODE_UNSPECIFIED
```

Comment:
Here + in docs: UNSPECIFIED values are typically not accepted by the REST API, and can be omitted.